apps/penxle.com/src/routes/editor/[permalink]/PublishMenuSearch.svelte

```diff
@@ -57,12 +59,16 @@
     }}
     use:outsideClickEvent
   >
-    <!-- on:outsideClick={() => (open = false)} -->
     <input
       class="rounded-1.5 bg-gray-50 py-2.5 pl-4 pr-4 text-14-r border border-gray-200 w-full"
       {placeholder}
       type="search"
       bind:value={query}
+      on:blur={() => {
+        if (query.length > 0) {
+          formEl.dispatchEvent(new Event('submit'));
```

Comment:
P2: 전송 메서드가 있는데 사용하는게 어떨까요?

```suggestion
          formEl.submit()
```

> https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit